### PR TITLE
Bump Flyway image to 8.5.10

### PR DESCRIPTION
**Pull request by tovbinm**
# Description

Bump Flyway image to 8.5.10 to allow arm64 support (Mac M1) 

image

+ Added docker multiarch build
https://www.docker.com/blog/multi-arch-images/

## Type of change
- New feature (non-breaking change which adds functionality)

# Checklist
- [x] Have you checked to there aren't other open Pull Requests for the same update/change?
- [x] Have you lint your code locally before submission?
- [x] Have you successfully run tests with your changes locally?

### CODE REVIEW

**Comment by tovbinm** [comment]
@ypc-faros can you please validate on your machine as well (since you have an arm64 chip)?

File: .github/workflows/ci-cd.yml [hunk 1]
```diff
@@ -102,7 +102,8 @@ jobs:
         run: |
           docker build . \
             -t "$INIT_LATEST_TAG" \
-            -t "$INIT_SHA_TAG"
+            -t "$INIT_SHA_TAG" \
+            --platform linux/amd64,linux/arm64,linux/arm/v7
```
Comment (tovbinm) [inline .github/workflows/ci-cd.yml:106]: @cjwooo @ted-faros - this is pretty awesome!

Comment (ted-faros) [reply .github/workflows/ci-cd.yml:106]: Agreed.  I noticed from above and [other](https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images) docs, the use of `docker buildx` command.  I'm guessing that is needed here too.

Comment (tovbinm) [reply .github/workflows/ci-cd.yml:106]: oh, checking...

File: .github/workflows/ci-cd.yml [hunk 2]
```diff
@@ -80,31 +80,33 @@ jobs:
       - name: Check out
         uses: actions/checkout@v2
 
+      - name: Docker meta
```
Comment (tovbinm) [inline .github/workflows/ci-cd.yml:83]: @cjwooo @ted-faros I switched to using docker official actions - meta, qemu, and buildx. 

How do we test this?! Should I try [`act`](https://github.com/nektos/act) ?

File: .github/workflows/ci-cd.yml (hunk 1 above)
Comment (tovbinm) [reply .github/workflows/ci-cd.yml]: Good point! Fixed.

**Comment by tovbinm** [comment]
@ypc-faros @cjwooo can I merge?

File: .github/workflows/ci-cd.yml (hunk 2 above)
Comment (cjwooo) [reply .github/workflows/ci-cd.yml:83]: Does this automatically generate the `latest` tag as well?

**Review by ypc-faros — APPROVED** [review]

File: .github/workflows/ci-cd.yml (hunk 2 above)
Comment (tovbinm) [reply .github/workflows/ci-cd.yml:83]: It can, I forgot to add it. Will do.

Comment (tovbinm) [reply .github/workflows/ci-cd.yml:83]: https://github.com/faros-ai/faros-community-edition/pull/125

**Pull request merged**